locale/coa/ca/French-Quebec_General.xml

```diff
@@ -0,0 +1,893 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration xmlns="http://ledgersmb.org/xml-schemas/configuration">
+  <gifi-list>
```

Comment:
TODO: Handle GIFI validity periods.

Reply:
Let's make an issue for that and discuss what that would take in that issue.